catalogs/bundled-catalog/src/main/java/com/datastrato/gravitino/catalog/property/PropertyConverter.java

```diff
@@ -40,13 +38,6 @@ public abstract class PropertyConverter {
    */
   public abstract Map<String, String> engineToGravitinoMapping();
 
-  /**
-   * Get the property metadata for the catalog. for more please see {@link
-   * HiveTablePropertiesMetadata#propertyEntries()} or {@link
-   * com.datastrato.gravitino.catalog.lakehouse.iceberg.IcebergTablePropertiesMetadata#propertyEntries()}
-   */
-  public abstract Map<String, PropertyEntry<?>> gravitinoPropertyMeta();
-
   Map<String, String> revsereMap(Map<String, String> map) {
```

Comment:
@FANNG1 can you please fix this typo "reverse".

Reply:
done